patches/@eth-optimism+contracts+0.1.11.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/node_modules/@eth-optimism/contracts/build/contracts/OVM/bridge/tokens/Abs_L2DepositedToken.sol b/node_modules/@eth-optimism/contracts/build/contracts/OVM/bridge/tokens/Abs_L2DepositedToken.sol
+index dd56f40..10e5dc9 100644
+--- a/node_modules/@eth-optimism/contracts/build/contracts/OVM/bridge/tokens/Abs_L2DepositedToken.sol
++++ b/node_modules/@eth-optimism/contracts/build/contracts/OVM/bridge/tokens/Abs_L2DepositedToken.sol
+@@ -45,6 +45,7 @@ abstract contract Abs_L2DepositedToken is iOVM_L2DepositedToken, OVM_CrossDomain
+     constructor(
+         address _l2CrossDomainMessenger
+     )
++        internal
```

Comment:
Optimism has a bug, so I needed to add constructor visibility modifiers to be able to compile with v0.6.

Reply:
I was not able to get Optimism contracts to compile on v0.6, so I migrated our contracts to v0.7 to avoid these compatibility issues.